### ISSUE TITLE
Conditional configuration should have hostname predefined.

### ISSUE THREAD

**Issue by lkarsten** [acassen/keepalived#769]
Hi.

I'm using keepalived 1.3.2 on Ubuntu 17.10, and I'm trying to use the conditional configuration option in `keepalived.conf`.

My main goal is not to have to configure more than strictly necessary, as every file that is changed must be put into revision control and distributed using the configuration control system.

The idea was to use something like this:
```
global defs {
@main   router_id main
@backup   router_id backup
}

vrrp_instance vrrpinstance_foo {
@main    state MASTER
@main    priority 100
@backup  state BACKUP
@backup priority 80
}
``` 
however, that only moves the problem to the systemd service file, which defines the startup arguments. Now I need to make sure that is updated and is different on the two hosts.

Would it be possible to have the local hostname value defined as a key by default?

That way I could do:
```
global defs {

@lb1   router_id main
@lb2  router_id backup
}
```
without having to modify the service file.

There is probably variable substitution available in the service file, which can be sent as the `-i` argument, but that leads me back to having to distribute the `keepalived.service` file.

**Comment by pqarmitage** [comment]
Commit d2901e49 implemented the functionality you are requesting, and although the commit is dated 4 December 2016 it appears it wasn't merged until 4 October 2017, and so was first included in v1.3.8.

**Comment by lkarsten** [comment]
Hi, and thanks for the quick response. 

Excellent news, I'll fetch the new version and give that a spin.

Thanks for writing/maintaining keepalived!
